### PR TITLE
Ot 147 47 Update News Entity

### PR DESCRIPTION
**Pull request by walterio123**
**Summary**

- Creation of the Controller route to update News Entity.
- Creation of the method in the service to update a News entity.
- Add the route to the security so that update the entity is only for the Admin role.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/auth/config/SecurityConfiguration.java [hunk 1]
```diff
@@ -75,6 +75,7 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
 
                 //News
                 .antMatchers(GET, "/news/{id}").hasAuthority("ADMIN")
+                .antMatchers(POST, "/news").hasAuthority("ADMIN")
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/auth/config/SecurityConfiguration.java:78]: The route should be "/news/{id} I believe

File: src/main/java/com/alkemy/ong/auth/config/SecurityConfiguration.java [hunk 2]
```diff
@@ -76,6 +76,7 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
                 //News
                 .antMatchers(GET, "/news/{id}").hasAuthority("ADMIN")
                 .antMatchers(POST, "/news").hasAuthority("ADMIN")
+                .antMatchers(PUT, "/news").hasAuthority("ADMIN")
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/auth/config/SecurityConfiguration.java:79]: The route should be "news/{id}" I believe

**Comment by DanteIturri** [comment]
resolve branch conflicts

**Comment by anavduo** [comment]
Nice Job!

**Pull request merged**